src/index.js

```diff
@@ -1,6 +1,22 @@
-export * from './components'
-export * from './icons'
-export * from './elements'
-
-// variables
-export { colors, breakpoints } from './variables'
+// Components
```

Comment:
was this manual? Can we add a script similar to the image one for this? That would be awesome